pages/user/index.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from 'react';
+import Image from 'next/image';
+
+async function retrieveUser(parentObjectId) {
+  const response = await fetch('http://34.210.145.64:8000/user/retrieve-user/' + parentObjectId);
+  return response; // Note: Can I make this response.json()?
+}
+
+function Event() {
+
+  const [user, setUser] = useState();
+
+  const parentObjectId = '6418ea4bfa9f8c7a6804cf77';
+
+  useEffect(() => {
+    retrieveUser(parentObjectId).then(
+      result => result.json()).then(
+        data => {
+          setUser(data.user);
+          console.log(data.user);
+        }
+      );
+  }, []);
+
+  return (
+    <div>
+      <div>
+        <div className="flex flex-row justify-between mx-10">
+          <div className='w-1/4'>Make group info here</div>
+          <div className='w-1/2 flex flex-col gap-5'>
+            <div className='flex flex-col'>
+              <span>{user?.name}</span>
+              <span>Organizer</span>
+              <span>Give member a custom title</span>
+            </div>
+            <div className='flex flex-row'>
+              <span className='flex flex-row'>
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="w-6 h-6">
```

Comment:
Does `strokeWidth` throw errors?
It might have been `stroke-width` (cabob case that was throwing console errors)